rawx-apache2/src/rawx_repo_core.c

```diff
@@ -479,7 +479,10 @@ chunk_verify_checksum(dav_resource *resource, request_rec *r)
 		EXTRA_ASSERT(resource->info);
 		struct chunk_textinfo_s *p = &resource->info->chunk;
 
-		EXTRA_ASSERT(p->chunk_hash);
+		if (! p->chunk_hash) {
+			g_checksum_free(md5);
+			return APR_EMISMATCH;
```

Comment:
`user.grid.chunk.hash` is a mandatory xattr. So if the hash isn't available, it's not a `Precondition Failed`. We should return an `Internal Server Error` (for example).